wpeverest/user-registration
# Feature - Slot booking

### All Submissions:


### Changes proposed in this Pull Request:

This PR is introducing the new slot booking feature. 

### ### Dependency PR's:
Pro - https://github.com/wpeverest/user-registration-pro/pull/292/files#
Advanced fields - https://github.com/wpeverest/user-registration-advanced-fields/pull/65

### How to test the changes in this Pull Request:

1. Create a form and drag the date and time picker field.
2. Goto the setting of each field enable the slot booking and choose the target date and timepicker field accordingly.
3. Preview the form and choose the date and time submit the form and again try to choose the same date and time and verify the error message.

### Types of changes:


### Other information:


### Changelog entry

>Feature - Slot booking.
